core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -140,7 +140,8 @@ $(document).ready(function() {
             (attempt.hasOwnProperty("attemptId") ? attempt["attemptId"] + "/" : "") + "logs";
           attempt["durationMillisec"] = attempt["duration"];
           attempt["duration"] = formatDuration(attempt["duration"]);
-          var app_clone = {"id" : id, "name" : name, "version": version, "num" : num, "attempts" : [attempt]};
+          var app_clone = {"id" : id, "name" : name, "version": version, "num" : num, "attempts" : [attempt],
```

Comment:
As I see as you introduced the `attemptId` instead of `num` now in the `historypage-template.html` there is no more usage of the `num` property (view data property). If it is not used can you please delete it.

Reply:
Ok, i will remove it.